src/embezzler.ts

```diff
@@ -30,8 +36,10 @@ import {
   SourceTerminal,
   sum,
 } from "libram";
+import { acquire } from "./acquire";
 import { Macro } from "./combat";
 import { globalOptions } from "./globalvars";
+import { baseMeat, WISH_VALUE } from "./lib";
```

Comment:
I'm not entirely sure I understand the utility of making WISH_VALUE a variable, but I guess it does come up a lot

Reply:
The advantage is it reduces one thing a person has to think about. Instead of seeing 50000 and having to remember it is the value of a wish, that is now explicit.